### PR TITLE
feat: wrap with semantic widget for talkback

### LINKED ISSUE #500: Accessibility: unlabeled buttons while using talkback
When using the Talkback screen reader for android the highlighted buttons and elements either only read as "unlabeled" or "button". The screen reader can be enabled in android settings > accessibility

stack 1
stack 2
stack 3
stack 4
stack 5
stack 6
stack 7
stack 8
stack 9
stack 10
stack 11
stack 12
stack 13
stack 14
stack 15
stack 16
stack 17
stack 18
stack 19
stack 20
stack 21
stack 22
stack 23
stack 24
stack 25
stack 26
stack 27
stack 28
stack 29
stack 30
stack 31
stack 32
stack 33
stack 34
stack 35
stack 36
stack 37

### PR DESCRIPTION
**Pull request by detherminal**
This pr implements wrapping buttons and so many other widgets with semantic widget for talkback and labels them.

Closes #500

### CODE REVIEW

**Comment by julian-CStack** [comment]
Looks good to me. My only suggestion would be to use a more descriptive name such as `semanticsLabel` instead of `label` for custom widgets that pass the semantics label through such as https://github.com/cypherstack/stack_wallet/pull/505/files#diff-9ebb9d922fa60af59c78a62c8eefefcf143478c873c1cfca2fccfca1d1fdc3aeR69

File: lib/pages/receive_view/receive_view.dart [hunk 1]
```diff
@@ -143,6 +143,7 @@ class _ReceiveViewState extends ConsumerState<ReceiveView> {
               child: AspectRatio(
                 aspectRatio: 1,
                 child: AppBarIconButton(
+                  label: "Adress List Pop-up Button. Opens A Pop-up For Adress List Button.",
```
Comment (sneurlax) [inline lib/pages/receive_view/receive_view.dart:146]: Spelling: "Address"

**Review by sneurlax — CHANGES_REQUESTED** [review]
LGTM, besides some spelling mistakes: "Openes" (Opens) and "Adresses" (Addresses).  Not sure if every word needs to be capitalized for talkback or if it matters (would it ever be seen?)

Does this close all of the examples in #500, most, some, etc?  We can merge this is and have a tester tap through for a list of everything we might want talkback on?

Tests just need to be rerun on our side with our secrets.  I can merge your staging to cypherstack:stack_wallet#detherminal and we can PR that to staging to (probably) pass tests

**Comment by detherminal** [comment]
Hey @sneurlax I changed ```label```s to ```semanticsLabel```s and corrected misspelling. You might want to check it out.

**Review by julian-CStack — APPROVED** [review]

**Review by sneurlax — APPROVED** [review]
Approved, I just don't know if it addresses *every* button/widget listed in the issue

**Pull request merged**